### PR TITLE
Pamm 2020 9 21

### PR DESCRIPTION
**Pull request by slacAWallace**
Updating to newest libraries.

### CODE REVIEW

File: lcls-plc-lfe-optics/lfe_optics/lfe_optics.plcproj [hunk 1]
```diff
@@ -158,13 +158,13 @@
     <LibraryReference Include="LCLS General,2.4.1,SLAC">
       <Namespace>LCLS_General</Namespace>
     </LibraryReference>
-    <LibraryReference Include="lcls-twincat-motion,1.1.0,SLAC">
+    <LibraryReference Include="lcls-twincat-motion,1.3.3,SLAC">
       <Namespace>lcls_twincat_motion</Namespace>
     </LibraryReference>
-    <LibraryReference Include="lcls-twincat-optics,0.0.1,SLAC">
+    <LibraryReference Include="lcls-twincat-optics,0.0.0,SLAC">
```
Comment (jsheppard95) [inline lcls-plc-lfe-optics/lfe_optics/lfe_optics.plcproj:164]: Did we mean to switch back to the dev version? Were changes made to `lcls-twincat-optics` during the last PAMM?

**Review by jsheppard95 — COMMENTED** [review]
This looks good/ready to merge, just wondering about the `lcls-twincat-optics` version.

File: lcls-plc-lfe-optics/lfe_optics/lfe_optics.plcproj (hunk 1 above)
Comment (slacAWallace) [reply lcls-plc-lfe-optics/lfe_optics/lfe_optics.plcproj:164]: Yes, I moved optics to 000 to unlock the pinned motion version in there. When you pin libraries inside libraries it creates duplicate symbols which breaks pytmc builds.

**Comment by ZLLentz** [comment]
Suppose we were going to recover post-power-outage. Would we use this PR, or master?

**Comment by ZLLentz** [comment]
closing in favor of #16

**Pull request closed without merging**